### PR TITLE
fix: Removed files accidentally added on PR 27489

### PR DESCRIPTION
**Pull request by RandellDawson**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Earlier when I was testing a guide PR which I resolved a conflict, I wanted to view it locally, to validate how it looked. To do this, I added the php folder inside the guide folder to the mock-guide folder. I have done this numerous times in the past and always remember to delete anything I add to the mock-guide before pushing back to the repo.

Well, this time I did not and this PR removes those 76 files which should not have been added to the mock-guide folder.

Looking for a quick review/merge of this PR to reverse this mistake.

### CODE REVIEW

File: mock-guide/english/php/functions/files/file-writing/index.md [hunk 1]
```diff
@@ -1,52 +0,0 @@
----
-title: File Writing
```
Comment (ValeraS) [inline mock-guide/english/php/functions/files/file-writing/index.md]: @RandellDawson Do not delete this file. This one is used in tests.

**Review by ValeraS — CHANGES_REQUESTED** [review]

File: mock-guide/english/php/functions/files/file-writing/index.md (hunk 1 above)
Comment (RandellDawson) [reply mock-guide/english/php/functions/files/file-writing/index.md]: Ah, now I understand why it is failing.  I could not figure out for the life of me, what the issue was.

Comment (RandellDawson) [reply mock-guide/english/php/functions/files/file-writing/index.md]: What is the easiest way for me to just add the one file back?

Comment (RandellDawson) [reply mock-guide/english/php/functions/files/file-writing/index.md]: I figured it out.

**Review by ValeraS — APPROVED** [review]

**Pull request merged**